tests/unit/modules/test_npm.py

```diff
@@ -38,39 +39,49 @@ def setup_loader_modules(self):
 
     def test_install(self):
         '''
-        Test if it install an NPM package.
+        Test if it installs an NPM package.
         '''
         mock = MagicMock(return_value={'retcode': 1, 'stderr': 'error'})
         with patch.dict(npm.__salt__, {'cmd.run_all': mock}):
             self.assertRaises(CommandExecutionError, npm.install,
                               'coffee-script')
 
-        mock = MagicMock(return_value={'retcode': 0, 'stderr': 'error',
-                                       'stdout': '{"salt": ["SALT"]}'})
+        # This is at least somewhat closer to the actual output format.
+        # Older versions have an outer array; newer ones have a hash
+        mock_json_out = textwrap.dedent('''\
+        [
+          {
+            "salt": "SALT",
+          }
+        ]''')
+
+        # Successful run, expected output format
+        mock = MagicMock(return_value={'retcode': 0, 'stderr': '',
+                                       'stdout': mock_json_out})
         with patch.dict(npm.__salt__, {'cmd.run_all': mock}):
-            mock_err = MagicMock(return_value='SALT')
-            with patch.object(salt.utils.json, 'loads', mock_err):
```

Comment:
I'm not sure if dropping this second patch is better or not, seems like it shouldn't be needed here?